<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ADM_cadastro_editar</class>
 <widget class="QDialog" name="ADM_cadastro_editar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Editando o Cadastro:</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Tipo de cadastro:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_tipo">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <bold>true</bold>
          </font>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LayoutDirection::RightToLeft</enum>
         </property>
         <property name="text">
          <string>Id:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_id">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_nome">
         <property name="text">
          <string>Nome</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txt_NovoNome"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_username">
         <property name="text">
          <string>Username</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txt_NovoUsername"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_senha">
         <property name="text">
          <string>Senha</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txt_NovoSenha"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_especialidade">
         <property name="text">
          <string>Especialidade</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txt_NovoEspecialidade">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="accessibleDescription">
          <string/>
         </property>
         <property name="text">
          <string>Não é possível mudar a especialidade</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ADM_cadastro_editar</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ADM_cadastro_editar</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
